lib/matplotlib/style/core.py

```diff
@@ -84,6 +85,73 @@ def use(style):
                 raise IOError(msg % style)
 
 
+def get_substyles(styles):
```

Comment:
Well, this will indeed look for child stylesheet, but also for children of those children. As you know, the `matplotlib.style.use` function takes a string *or* a list. This will scan each style defined in this list and append after each style its children in the list. 
I did this to be able to create stylesheet from multiple smaller stylesheets, to be able to create full styles easily